src/elements/content-preview/__tests__/PreviewNavigation.test.js

```diff
@@ -18,110 +19,284 @@ const deeplinkedMetadataHistoryMock = {
         });
 
-        test('should render left navigation correctly from metadata deeplinked URL ', () => {
+        test('should render left navigation correctly from metadata deeplinked URL', async () => {
             const onNavigateLeftMock = jest.fn();
-            const wrapper = getWrapper({
+            const user = userEvent();
+
+            renderComponent({
                 currentIndex: 2,
                 historyMock: deeplinkedMetadataHistoryMock,
                 onNavigateLeft: onNavigateLeftMock,
             });
 
-            expect(wrapper.find('PlainButton')).toHaveLength(1);
-            wrapper.find('PlainButton').simulate('click');
+            const leftButton = screen.getByTestId('preview-navigation-left');
+            expect(leftButton).toBeInTheDocument();
 
-            expect(deeplinkedMetadataHistoryMock.push).toBeCalledTimes(1);
-            expect(deeplinkedMetadataHistoryMock.push).toBeCalledWith('/metadata/filteredTemplates/123,124');
+            await user.click(leftButton);
+
+            expect(deeplinkedMetadataHistoryMock.push).toHaveBeenCalledTimes(1);
+            expect(deeplinkedMetadataHistoryMock.push).toHaveBeenCalledWith('/metadata/filteredTemplates/123,124');
             expect(onNavigateLeftMock).toHaveBeenCalled();
         });
     });
+
+    describe('when routerDisabled is true', () => {
+        test('should render correctly without router', () => {
+            renderComponentWithoutRouter({ currentIndex: 1 });
+
+            expect(screen.getByTestId('preview-navigation-left')).toBeInTheDocument();
+            expect(screen.getByTestId('preview-navigation-right')).toBeInTheDocument();
+        });
+
+        test('should call internalSidebarNavigationHandler when left navigation button is clicked', async () => {
+            const mockNavigationHandler = jest.fn();
+            const mockInternalSidebarNavigation = {
+                sidebar: ViewType.ACTIVITY,
+                activeFeedEntryType: FeedEntryType.COMMENTS,
+                activeFeedEntryId: '123',
+            };
+            const onNavigateLeftMock = jest.fn();
+            const user = userEvent();
+
+            renderComponentWithoutRouter({
+                currentIndex: 1,
+                internalSidebarNavigationHandler: mockNavigationHandler,
+                internalSidebarNavigation: mockInternalSidebarNavigation,
+                onNavigateLeft: onNavigateLeftMock,
+            });
+
+            const leftButton = screen.getByTestId('preview-navigation-left');
+            await user.click(leftButton);
+
+            expect(mockNavigationHandler).toHaveBeenCalledTimes(1);
+            expect(mockNavigationHandler).toHaveBeenCalledWith({ sidebar: ViewType.ACTIVITY });
+            expect(onNavigateLeftMock).toHaveBeenCalledTimes(1);
+        });
+
+        test('should call internalSidebarNavigationHandler when right navigation button is clicked', async () => {
```

Comment:
we can combine this test and `should call navigation handler with metadata deeplinks when right navigation button is clicked`, the setup is the same only one more expect is needed
the same goes for `left navigation`